griptape/utils/stream.py

```diff
@@ -53,11 +53,14 @@ def run(self, *args) -> Iterator[TextArtifact]:
         t.join()
 
     def _run_structure(self, *args):
+        from griptape.drivers import LocalEventListenerDriver
+
         def event_handler(event: BaseEvent):
             self._event_queue.put(event)
 
         stream_event_listener = EventListener(
-            event_handler, event_types=[CompletionChunkEvent, FinishPromptEvent, FinishStructureRunEvent]
```

Comment:
Now I see why you overrode `publish_event` in `LocalEventListenerDriver`

Reply:
🧵